src/main/java/homework_1/CharCount.java

```diff
@@ -0,0 +1,19 @@
+package homework_1;
+
+public class CharCount {
+    public static void main(String[] args) {
+        if (args.length == 0) {
+            System.out.println("There is no parameters, exit program.");
+        } else {
+
+            System.out.println("Count of Char's in words:");
+            for (String arg : args) {
+                if (arg.equals("error")) {
+                    System.err.println("Alarm!");
```

Comment:
Doesn't work as expected
![image](https://user-images.githubusercontent.com/58872509/125207756-af5cd300-e296-11eb-99c1-edfedcdd4c0f.png)

Reply:
Still the same